soh/soh/Enhancements/game-interactor/GameInteractionEffect.cpp

```diff
@@ -350,10 +350,10 @@ namespace GameInteractionEffect {
         }
     }
     void OneHitKO::_Apply() {
-        GameInteractor::State::OneHitKOActive = 1;
+        GameInteractor::State::OneHitKOActive = true;
     }
     void OneHitKO::_Remove() {
-        GameInteractor::State::OneHitKOActive = 0;
+        GameInteractor::State::OneHitKOActive = false;
     }
```

Comment:
Isn't this going to cause issues considering `GameInteractor::State::OneHitKOActive` is used in C files? That's why I've used 0 and 1 for true/false so far.

Reply:
Those still return that class as a uint8_t though. Does the conversion between the bool and uint8_t work properly? Just mentioning this because we ran into this exact issue several times before.